src/hb-sanitize.hh

```diff
@@ -116,13 +116,17 @@
 #ifndef HB_SANITIZE_MAX_SUBTABLES
 #define HB_SANITIZE_MAX_SUBTABLES 0x4000
 #endif
+#ifndef HB_SANITIZE_MAX_DEPTH
+#define HB_SANITIZE_MAX_DEPTH 2500
```

Comment:
Move this into COLR then. And I feel like 128 should be more than enough. That's a lot of stack!

There's already `COLRV1_MAX_NESTING_LEVEL`. Reuse it. Rename to `HB_COLR_MAX_NESTING_LEVEL` please.

Reply:
Ah hadn't noticed that constant. Updated to reuse it.